.travis.yml

```diff
@@ -1,6 +1,8 @@
 {
   "language": "node_js",
-  "node_js": "8",
+  "node_js": 
+    - "8"
+    - "10",
```

Comment:
We can experiment with other versions of nodejs in the future, but these are good for now.